src/webapp/features/rabble_logo/rabbleLogo.module.css

```diff
@@ -0,0 +1,183 @@
+@media only screen and (max-width: 550px) {
```

Comment:
Tested on ios, the alignment is a tiny bit off - a bit to the right, and cut off by that dumb bar at the bottom:

![ios-rabble](https://user-images.githubusercontent.com/4634442/104131620-394dd280-532c-11eb-9eea-cef59311216e.png)

For a little to the right, i'm sure there's a CSS fix.

For the bar at the bottom, let's consider making the cubes a bit smaller on mobile?

Also, this points at our need to get you a way to test on IOS... I have an idea, we can discuss on our next call.

Reply:
interesting that chrome doesn't include the utility bar to check screen size. Noted!

Reply:
should be super easy once I can preview in xcode as all of those measurements are based in VH and %, no probs to make it smaller and more central